tests/orchestrator/test_sequential_orchestrator.py

```diff
@@ -268,6 +299,33 @@ def test_user_code_raises_exception():
         expected_error_str = f"{error_msg}{error_label}{state_str}"
         assert expected_error_str == error_str
 
+def test_user_code_raises_exception_with_pystein():
+    context_builder = ContextBuilder('test_simple_function')
```

Comment:
can we check the metadata info(like triggers, bindings, number of functions) in the dfapp instance in these tests?

Reply:
for every trigger/binding type(like eventhub, blob, cosmosdb,etc) we are writing two types of unit tests
1. unit test for low level metaclass similar to the link below: (to make sure binding type and get_binding_repr match with function.json definition)
https://github.com/Azure/azure-functions-python-library/blob/dev/tests/decorators/test_eventhub.py

2. tests for decorators: 
<img width="443" alt="image" src="https://user-images.githubusercontent.com/12449837/203163113-52e2d04b-a242-47a8-85a9-f4f5686a73bb.png">
https://github.com/Azure/azure-functions-python-library/blob/dev/tests/decorators/test_decorators.py
test for each durable pystein decorator, the generated function.json file is correct